src/python/grpcio/grpc/__init__.py

```diff
@@ -424,6 +424,23 @@ def __init__(self, credentials):
         self._credentials = credentials
 
 
+class ServerCertificateConfig(object):
+    """A certificate config for use with an SSL-enabled Server, e.g., can
+    be returned in the certificate config fetching callback.
+
+    This class has no supported interface -- it exists to define the
+    type of its instances and its instances exist to be passed to
+    other functions.
+
+    NOTE: do not re-use instances of this class, i.e, always create a
```

Comment:
This seems like a usability problem. What's the reason for this restriction? What problems arise if it is violated?

Reply:
it's because at https://github.com/grpc/grpc/pull/13188/files#diff-96390deae069919cdf8b9d18d2e92687R46 the `c_cert_config` member of `ServerCertificateConfig` is passed to C-core, which takes over the ownership of the pointer, so i have to null it out. so, we could improve usability--i.e., allow re-use--then we have to make copies of `c_cert_config`, perhaps with a new API in C-core such as `cert_config_dup()` or we have to manually recreate `c_cert_config` in cython after passing it to C-core.
suggestions?

Reply:
sounds good. i'll make cython layer making the copies.